src/core/installAddon.js

```diff
@@ -234,6 +234,7 @@ type WithInstallHelpersInternalProps = {|
   currentVersion: AddonVersionType | null,
   dispatch: DispatchFunc,
   userAgentInfo: UserAgentInfoType,
+  version: AddonVersionType | null,
```

Comment:
This relies on [`...passThroughProps`](https://github.com/mozilla/addons-frontend/blob/94f1e6d641a4589f028797725998cb6204468f88/src/core/installAddon.js#L550) which I don't like because it's implicit and slightly magical.

Can you just get `version` from `ownProps.version` in the mapper and map it directly as an internal prop? That would make it explicit.

https://github.com/mozilla/addons-frontend/blob/94f1e6d641a4589f028797725998cb6204468f88/src/core/installAddon.js#L582-L586

You might even be able to adjust the value of `currentVersion` in that mapper like `currentVersion = ownProps.version || getVersionById(...)`. I haven't fully thought that through but it could make this patch more robust since everything internally would only have to reference one property.